package.json

```diff
@@ -4,7 +4,7 @@
   "description": "config and resource loader",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "tape test.js"
```

Comment:
@anuragCES since there is no need for babel as I've mentioned in the comment for `.babelrc` this should be OK.